app/controllers/projects_controller.rb

```diff
@@ -217,4 +218,8 @@ def find_project_by_id
     def redirect_to_slug
       redirect_to @project if @project
     end
+
+    def username_arr
+      @usernamearray = User.pluck(:name).compact.to_json
```

Comment:
I'll prefer `@username_array` is more understandable. Also for the name of the method, `username_array` instead of `username_arr`

Reply:
done